featurebyte/api/base_table.py

```diff
@@ -68,6 +68,27 @@ def info(self) -> ColumnInfo:
         column_info = next(col for col in self.parent.columns_info if col.name == self.name)
         return cast(ColumnInfo, column_info)
 
+    @property
+    def cleaning_operations(self) -> List[CleaningOperation]:
+        """
+        Cleaning operations applied to the column of the table.
+
+        Returns
+        -------
+        List[CleaningOperation]
+            List of cleaning operations applied to the column of the table.
+
+        Examples
+        --------
+        >>> event_table = catalog.get_table("GROCERYINVOICE")
+        >>> event_table["GroceryInvoiceGuid"].cleaning_operations
+        []
```

Comment:
think xavier prefers if we could have an example that displays actual cleaning operations, instead of an empty list

can refer to the change i made recently - https://github.com/featurebyte/featurebyte/commit/f3485f39d8de0c7e64cbb642b7e3e2cffc173b30

Reply:
Got it, will update the example here.